src/Components/SmartComponents/SystemCves/SystemCveTable.js

```diff
@@ -58,9 +58,9 @@ const SystemCvesTableWithContext = ({ context, header, entity }) => {
 
     const rows = !isEmpty ? cves.data
     .map(cve => (selectedCves.includes(cve.id) && { ...cve, selected: true }) || cve)
-    .map(cve => {
-        const current = openedCves.find(opened => opened.id === cve.id);
-        return ({ ...cve, isOpen: current && current.isOpen || false });
+    .map((cve, index) => {
+        const current = index % 2 === 0 ? openedCves.find(opened => opened.id === cve.id) || false : { isOpen: undefined };
```

Comment:
do we really need { isOpen: undefined } ?
I am not sure if it works but I think `index % 2 === 0 && openedCves.find(opened => opened.id === cve.id) || false`

Reply:
Yes! This is actually the bug, false is different than undefined:
isExpanded undefined means there should be no arrow,
isExpanded false means there should be an arrow facing downward,
isExpanded true means there should be an arrow facing upward.

This is pretty unintuitive at first, I was confused too.